config.md

```diff
@@ -535,3 +535,23 @@ Available kinds could be found [here](features#kinds) in the Kind column.
 Specify semantic token modifiers that clangd should not send to client.
 
 Available modifiers could be found [here](features#modifiers) in the Modifier column.
+
+## Documentation
+{:.v21}
```

Comment:
This needs to be v22 (https://github.com/llvm/llvm-project/commit/51994891e3a28a999e30a915d9ac5c2102073576 is not present on the 21 branch), and as it's the first option to be added in v22, the [styles](https://github.com/llvm/clangd-www/blob/1b1fd55a92d7ab4f177e6118ab5e370e897a3a80/styles.css#L170-L200) need to be adjusted for it.